cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,52 @@
 describe('', () => {
```

Comment:
The describe() function should be named as the module name that is tested in this block. Add this